.eslintrc.yml

```diff
@@ -0,0 +1,16 @@
+env:
+  browser: true
+  es2020: true
+  node: true
+extends:
+  - google
+parserOptions:
+  ecmaVersion: 12
+ignorePatterns: "library_*.js"
```

Comment:
is this meant to ignore the generated code? then there are also pre/postamble e tc.

Reply:
So far I'm just running this on a single file.. so this pattern is kind of redundant for now.